desktop/hardened-desktop/admin-controls/configure-ac.md

```diff
@@ -0,0 +1,144 @@
+| `proxy` | It is used for `http` and `https`.  If the port is custom, specify it in the property. |
+| `enhancedContainerIsolation`  | If `value` is set to true, Docker Desktop runs all containers as unprivileged, via the Linux user-namespace, prevents them from modifying sensitive configurations inside the Docker Desktop VM, and uses other advanced techniques to isolate them. For more information, see [Enhanced Container Isolation](../enhanced-container-isolation/index.md). |
+|`useWindowsContainers` | <span class="badge badge-info">Windows only</span> If `value` is set to true, it switches Docker Desktop to toggle the Docker CLI to talk to the Windows daemon, enabling Windows containers. If false, switches Docker Desktop to toggle the Docker CLI to talk to the Linux daemon, enabling Linux containers. This overrides anything that may have been set at installation using the `--no-windows-containers` flag.|
+| `linuxVM` | Parameters and settings related to Linux VM options - grouped together here for convenience. |
+| &nbsp; &nbsp; &nbsp; &nbsp;`wslEngineEnabled`  |<span class="badge badge-info">Windows only</span> If `value` is set to true, Docker Desktop uses the WSL 2 based engine. This overrides anything that may have been set at installation using the `--backend=<backend name>` flag. It is also incompatible with Enhanced Container Isolation. See [Known issues](faq.md) for more information.|
+| &nbsp;&nbsp; &nbsp; &nbsp;`dockerDaemonOptions`|If `value` is set to true, it overrides the options in the Linux daemon config file. See the [Docker Engine reference](../../../engine/reference/commandline/dockerd/#daemon-configuration-file). |
+| &nbsp;&nbsp; &nbsp; &nbsp;`vpnkitCIDR` |Overrides the network range used for vpnkit DHCP/DNS for `*.docker.internal`  |
+| `windowsContainers` | Parameters and settings related to `windowsContainers` options - grouped together here for convenience.                  |
+| &nbsp; &nbsp; &nbsp; &nbsp;`dockerDaemonOptions` | Overrides the options in the linux daemon config file. See the [Docker Engine reference](../../../engine/reference/commandline/dockerd/#daemon-configuration-file).|                                |
+|`disableUpdate`|If `value` is set to true, checking for and notifications about Docker Desktop updates is disabled.|
+|`analyticsEnabled`|If `value` is set to false, Docker Desktop does not send usage statistics to Docker. |
+
+## Step three: Re-launch and re-authenticate
+>Note
+>
+>Administrators should test the changes made through the `admin-settings.json` file locally to see if the settings work as expected.
+
+Once you have created and configured `admin-settings.json`, developers need to quit Docker Desktop through the Whale menu, and then relaunch Docker Desktop and sign in to receive the changed settings. Selecting **Restart** from the Whale menu isn't sufficient as it only restarts some components of Docker Desktop.
```

Comment:
We need to be careful here as this slightly nuanced.

For settings to take effect:

1. On a new install, developers just have to open Docker Desktop and authenticate to their org

2. On an existing install, developers have to quit (via the whale menu) and restart Docker Desktop. They must also ensure they are signed in to their org. (but they don't have to reauthenticate to their org)


@ctalledo please keep me honest here !

Reply:
btw - im not saying that we should structure it as a numbered list, just trying to flag how this works for either use case.